restricted_site_access.php

```diff
@@ -955,7 +955,7 @@ public static function admin_head() {
 		$screen->add_help_tab(
 			array(
 				'id'      => 'restricted-site-access',
-				'title'   => esc_html_x( 'Restricted Site Acccess', 'help screen title', 'restricted-site-access' ),
+				'title'   => esc_html_x( 'Restricted Site Access', 'help screen title', 'restricted-site-access' ),
```

Comment:
Although this change is unrelated to the issue at hand, I included it so translators can manage two identical strings in once change.